tsconfig.json

```diff
@@ -20,7 +20,8 @@
       "@ngneat/transloco-persist-lang": ["./projects/ngneat/transloco-persist-lang/src/public-api.ts"],
       "@ngneat/transloco-persist-translation": ["./projects/ngneat/transloco-persist-translations/src/public-api.ts"],
       "@ngneat/transloco-messageformat": ["./projects/ngneat/transloco-messageformat/src/public-api.ts"],
-      "@ngneat/transloco-preload-langs": ["./projects/ngneat/transloco-preload-langs/src/public-api.ts"]
+      "@ngneat/transloco-preload-langs": ["./projects/ngneat/transloco-preload-langs/src/public-api.ts"],
+      "@ngneat/transloco-locale": ["./projects/ngneat/transloco-locale/src/public-api"]
```

Comment:
`.ts` is missing in path: `@ngneat/transloco-locale": ["./projects/ngneat/transloco-locale/src/public-api.ts"]`